tests/test_data/packages/fake_with_deps/setup.py

```diff
@@ -18,6 +18,5 @@
         "SQLAlchemy!=0.9.5,<2.0.0,>=0.7.8,>=1.0.0",
         "python-memcached>=1.57,<2.0",
         "xmltodict<=0.11,>=0.4.6",
-        "requests @ git+git://github.com/psf/requests@v2.25.1",
```

Comment:
I removed this line since it seemed to be useless IMO. I added it when I started working on my first PR in `pip-tools` for PEP 440 support and I had not yet written all the tests above.